docs/spec/nmt.md

```diff
@@ -78,70 +78,71 @@ The following enumerated cases explain potential outcomes when querying an NMT f
 
 #### Namespace Inclusion Proof
 
-When the queried namespace ID `nID` has corresponding items in the tree with root `T`, the query is resolved by a
+When the queried namespace ID `NS` has corresponding items in the tree with root `T`, the query is resolved by a
```

Comment:
```suggestion
When the queried namespace `NS` has corresponding items in the tree with root `T`, the query is resolved by a
```

Reply:
Addressed in https://github.com/celestiaorg/nmt/pull/209/commits/5946b91c6735ad609229942334882505f3e3ad79